public/static/js/projects.js

```diff
@@ -444,6 +484,37 @@ mciModule.controller('ProjectCtrl', function($scope, $window, $http, $location)
     return true;
   };
 
+  $scope.addSubscription = function() {
+      promise = addSubscriber($mdDialog, $scope.triggers);
+
+      $mdDialog.show(promise).then(function(data){
+          data.changed = true;
+          $scope.isDirty = true;
+          $scope.subscriptions.push(data);
+      });
+  };
+
+  $scope.editSubscription = function(index) {
+      promise = editSubscriber($mdDialog, $scope.triggers, $scope.subscriptions[index]);
+
+      $mdDialog.show(promise).then(function(data){
+          data.changed = true;
+          $scope.isDirty = true;
+          $scope.subscriptions[index] = data;
+      });
+  };
+
+  $scope.removeSubscription = function(index) {
+      if ($scope.subscriptions[index].id !== undefined) {
+          $scope.settingsFormData.delete_subscriptions.push($scope.subscriptions[index].id);
+      }
+      $scope.subscriptions = _.filter($scope.subscriptions, function(s, i) {
+          return index !== i;
+      });
+      $scope.isDirty = true;
+  };
+  $scope.show_build_break = true;
```

Comment:
Don't think this is needed - it's always true

Reply:
It's needed when rendering one of the partial templates (subscriptions_list.html). This variable makes the build-break notifications check box show up. I'm going to reuse this template when I implement the subscriptions menu for subscribing to versions/builds/etc.